arches/fusion.yaml

```diff
@@ -0,0 +1,74 @@
+top.cpu.core0:
+  fetch.params.num_to_fetch:   8
+  decode:
+    params:
+      num_to_decode: 8
+      fusion_enable: true
+      fusion_enable_register: 0xFFFFFFFF
+      fusion_max_latency:     8
+      fusion_match_max_tries: 1023
+      fusion_max_group_size:  8
+      fusion_summary_report:    fusion_summary.txt
+      fusion_group_definitions: [ arches/fusion/dhrystone.json ]
+
+  rename.params.num_to_rename: 8
+  rename.params.num_integer_renames: 64
+  rename.params.num_float_renames: 64
+  dispatch.params.num_to_dispatch: 8
+  rob.params.num_to_retire:    8
+  dcache.params:
+    l1_size_kb: 64
+
+top.cpu.core0.extension.core_extensions:
```

Comment:
Should be able to say:
```
include: big_core.yaml
```
and remove everything from here down.

Reply:
Corrected,

I do not know what I was doing before but it was not picking up fusion_enable state. Probably indentation issue. Fixed now